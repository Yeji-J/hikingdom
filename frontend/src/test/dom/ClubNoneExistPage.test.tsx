import React from 'react';
// import renderer from 'react-test-renderer';
// import { render, screen } from '@testing-library/react';
import { render } from 'test/render';
import ClubNoneExistPage from 'pages/club/ClubNoneExistPage';

describe('<ClubNoneExistPage />', () => {
  it('shows the current count state.', () => {
    const component = render(<ClubNoneExistPage />);
    expect(component).toMatchSnapshot();
  });
});