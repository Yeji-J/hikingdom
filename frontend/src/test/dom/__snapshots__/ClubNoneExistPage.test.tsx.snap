// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClubNoneExistPage /> shows the current count state. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="page light p-md page"
      >
        <div
          class="content"
        >
          <img
            class="image"
            src="mountain.png"
          />
          <span
            class="text"
          >
            킹덤에 가입하고 
            <br />
            나만의 킹덤을 꾸려보세요
          </span>
        </div>
        <div
          class="button"
        >
          <button
            class="button lg secondary"
          >
            가입 대기 목록
          </button>
          <button
            class="button lg primary"
          >
            모임 생성
          </button>
          <button
            class="button lg secondary"
          >
            모임 검색
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="page light p-md page"
    >
      <div
        class="content"
      >
        <img
          class="image"
          src="mountain.png"
        />
        <span
          class="text"
        >
          킹덤에 가입하고 
          <br />
          나만의 킹덤을 꾸려보세요
        </span>
      </div>
      <div
        class="button"
      >
        <button
          class="button lg secondary"
        >
          가입 대기 목록
        </button>
        <button
          class="button lg primary"
        >
          모임 생성
        </button>
        <button
          class="button lg secondary"
        >
          모임 검색
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
